### PR TITLE
Update wake_on_lan.markdown

### PR DESCRIPTION
**Pull request by vahaldor**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/wake_on_lan.markdown [hunk 1]
```diff
@@ -86,7 +86,7 @@ name:
   default: Wake on LAN
   type: string
 host:
-  description: The IP address or hostname to check the state of the device (on/off). If this is not provided, the state of the switch will be assumed based on the last action that was taken.
+  description: The IP address or hostname to check the state of the device (on/off). If this is not provided, the state of the switch will be assumed based on the last action that was taken. You have to allow (Echo Request - ICMPv4-In) in Windows firewall inbound rules to allow the integration ping your computer and read the switch status.
```
Comment (frenck) [inline source/_integrations/wake_on_lan.markdown:89]: We consider firewalling out of scope for our documentation. There are tons of firewalls available and even more targets for WoL, which can make this little description impossible to maintain.

Nevertheless, thanks for willing to contribute 👍 

../Frenck

**Comment by vahaldor** [comment]
So why not write it there using general, not windows related, words? People who are not techy as the developers do not know that ping is not enabled by default, I spent hours solving this problem. The documentation should contain reminders like enabling ping in the firewall is necessary, especially when there is no single word about ping being the method behind the "host" configuration variable.

Moreover the sentence: "It’s required that the binary ping is in your $PATH." is completely out of my understanding, while I consider myself to be at least a geeky guy.

**Comment by frenck** [comment]
> So why not write it there using general, not windows related, words? 

Because of the same reasoning I already stated.

**Comment by vahaldor** [comment]
I had two arguments, the second is: there is no single word about ping being the method behind the "host" configuration variable. This is not the first time something did not work for me because I did not know where to look. It is not a problem with firewall but about being even able to deduce why something does not work. You should focus on these tiny details in the documentation.

**Comment by frenck** [comment]
Thanks for your suggestion, however, I disagree with the addition made for the reasoning stated above.

**Comment by vahaldor** [comment]
Sorry, I don't understand. Do you think it is correctly written documentation, when you do not include what method is used to let me allow it? The documentation contains only requirement to add IP address to the configuration variable host. I did not use this integration for last three years until today, when I found out that ping does not work on my computer. So I tried and realized this is the problem.

Secondly, I mentioned that a sentence: "It’s required that the binary ping is in your $PATH." says nothing to even techy users. I think you should reconsider your stubborn position that the documentation is perfect. It is not and the experience of using it is what makes it better. Please!

**Comment by frenck** [comment]
> Sorry, I don't understand. Do you think it is correctly written documentation

Yes I do.

>  The documentation contains only requirement to add IP address to the configuration variable host.

Correct.

> did not use this integration for last three years until today, when I found out that ping does not work on my computer. So I tried and realized this is the problem.

Which is an external problem, there are tons of external factors that can be of influence. From "Make sure the device is connected to the network", Firewalls, and even routing between vlans and other setups. External factors are not in scope here, as we don't maintain those and have thousands of possible permutations.

> Secondly, I mentioned that a sentence: "It’s required that the binary ping is in your $PATH." says nothing to even techy users.

I agree, that could be removed (as is more of a left over from the old days), but that wasn't part of or related to this PR.

../Frenck

**Comment by vahaldor** [comment]
FRENCK I really adore your work and have spent with HA tons of hours. This is not an external problem. If I need to make something work, I need to know on what technology it is based to be able to debug the problem. This is the core of my plea. Forget the firewall, I deduce the problem with firewall immediately when I knew that state of the switch is dependent on ping.

**Comment by frenck** [comment]
I think you misspelled my name there.

> This is not an external problem. 

It is, it is a firewall, not Home Assistant.

>  I need to know on what technology it is based to be able to debug the problem.

I agree, but this is not our tech.

**Pull request closed without merging**